cli/tools/lint/linter.rs

```diff
@@ -225,14 +226,23 @@ fn apply_lint_fixes(
   if quick_fixes.is_empty() {
     return None;
   }
+
+  let mut import_fixes = HashSet::new();
   // remove any overlapping text changes, we'll circle
   // back for another pass to fix the remaining
   quick_fixes.sort_by_key(|change| change.range.start);
   for i in (1..quick_fixes.len()).rev() {
     let cur = &quick_fixes[i];
     let previous = &quick_fixes[i - 1];
-    let is_overlapping = cur.range.start < previous.range.end;
-    if is_overlapping {
+    // hack: deduplicate import fixes to avoid creating errors
+    if previous.new_text.trim_start().starts_with("import ") {
```

Comment:
This doesn't seem right. We should fix this before merging.

Reply:
Actually, I'm not sure about a way around this because for the lsp we want all the import fixes.